code/script.js

```diff
@@ -7,3 +7,14 @@ function toggle() {
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
 document.getElementById("section1").onclick = toggle
+
+document.getElementById("section2").onclick = toggle
+
+
+/*This will eventually be the toggle for accordion*/
+
+const btn = document.getElementById("button")
+
+btn.onclick = () => {
+  btn.classList.toggle("btn-dark")
+}
```

Comment:
What is this doing? What I can see it does only makes the button color. You can do it with css only actually! [Here](https://www.w3schools.com/cssref/sel_visited.asp) you read more about it :)